consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -361,11 +363,7 @@ impl<
         S: Supervisor<Index = View, PublicKey = C::PublicKey>,
     > Actor<E, C, D, A, R, F, S>
 {
-    pub fn new(
-        context: E,
-        journal: Journal<E>,
```

Comment:
We now create the journal inside of the actor because we must populate `codec_config` (often using internal types.